scripts/ufl-convert

```diff
@@ -93,10 +100,10 @@ for arg in args:
     # 1) Load forms
     ufl_data = load_ufl_file(uflfilename)
     forms = ufl_data.forms
-    #expressions = ufl_data.expressions # TODO: Allow rendering expressions without form stuff!
+    # expressions = ufl_data.expressions # TODO: Allow rendering expressions without form stuff!
 
     # Preprocess forms
-    form_datas = [compute_form_data(f) for f in forms]
+    form_datas = [compute_form_data(f, do_apply_geometry_lowering=True, complex_mode=options.complex) for f in forms]
```

Comment:
So we change the behaviour of preprocessing in this script, although by default we still return the non-preprocessed from.  Why is this change?

Reply:
I don't know. I didn't do it.